project/src/main/world/Overworld.tscn

```diff
@@ -280,15 +287,8 @@ codes = [ "bigfps" ]
 
 [node name="SettingsMenu" parent="Ui/OverworldUi" instance=ExtResource( 6 )]
 quit_text = "Save + Quit"
+[connection signal="timeout" from="World/Obstacles/Bort/MoveTimer" to="World/Obstacles/Bort" method="_on_MoveTimer_timeout"]
 [connection signal="timeout" from="World/Obstacles/Ebe/MoveTimer" to="World/Obstacles/Ebe" method="_on_MoveTimer_timeout"]
-[connection signal="chat_ended" from="Ui/OverworldUi" to="Ui/OverworldUi/TouchButtons" method="_on_OverworldUi_chat_ended"]
-[connection signal="chat_ended" from="Ui/OverworldUi" to="World/Camera" method="_on_OverworldUi_chat_ended"]
-[connection signal="chat_started" from="Ui/OverworldUi" to="Ui/OverworldUi/TouchButtons" method="_on_OverworldUi_chat_started"]
-[connection signal="chat_started" from="Ui/OverworldUi" to="World/Camera" method="_on_OverworldUi_chat_started"]
-[connection signal="showed_chat_choices" from="Ui/OverworldUi" to="World/Obstacles/Spira" method="_on_OverworldUi_showed_chat_choices"]
-[connection signal="chat_event_played" from="Ui/OverworldUi/ChatUi" to="Ui/OverworldUi" method="_on_ChatUi_chat_event_played"]
-[connection signal="pop_out_completed" from="Ui/OverworldUi/ChatUi" to="Ui/OverworldUi" method="_on_ChatUi_pop_out_completed"]
-[connection signal="showed_choices" from="Ui/OverworldUi/ChatUi" to="Ui/OverworldUi" method="_on_ChatUi_showed_choices"]
```

Comment:
why were these signals unhooked?

Reply:
Restored; sometimes the Godot editor randomly unhooks signals, sometimes it randomly unhooks a WHOLE lot at once. It seems like it's related to false positive circular dependencies.